LykkeApi2/Controllers/WatchlistsController.cs

```diff
@@ -74,7 +74,7 @@ public async Task<IActionResult> Create([FromBody] WatchListCreateModel model)
                 Id = Guid.NewGuid().ToString(),
                 Name = model.Name,
                 Order = model.Order,
-                AssetIds = model.AssetPairIds != null ? model.AssetPairIds.ToList() : model.AssetIds.ToList() //TODO: remove AssetIds at all
+                AssetIds = model.AssetPairIds.ToList()
```

Comment:
AssetsIds = AssetPairIds?

Reply:
It's an internal contract and has incorrect naming